docs/notebooks/getting_started.ipynb

```diff
@@ -0,0 +1,414 @@
+{
```

Comment:
<p>List being used in <code>results = model.fit("y ~ X1 + X2 + ['1|X3', '1|X4']", priors=priors)</code></p><p> </p><p>And I'm not sure this will work </p><pre class="ql-syntax" spellcheck="false">    # Specify model but don't build/sample just yet 
    model.fit('y ~ X1 + X3 + X4', group_specific='1|X2', run=False) 

    # Specify priors—produces same result as in Example 2 above 
    model.set_priors( 
        {'X1': 0.3}, 
        common=Prior('Normal', sd=100), 
        group_specific='wide' 
    ) 

    # Now sample 
    results = model.fit(draws=5000) 
</pre><p>I'll  try to check later and propose a solution</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/302/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>thanks</p><p> </p>

Reply:
<p>It does not work, but I have a fix for this. I think it's better to add it in a new PR, do you agree?</p>